frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -57,6 +57,9 @@ pods:
           mv hdfs {{TASKCFG_ALL_HDFS_VERSION}}/bin/ ;
           {{#SECURITY_KERBEROS_ENABLED}}
           export HADOOP_OPTS="-Djava.security.krb5.conf=$MESOS_SANDBOX/{{TASKCFG_ALL_HDFS_VERSION}}/etc/hadoop/krb5.conf $HADOOP_OPTS" &&
+          {{#TASKCFG_ALL_SECURITY_KERBEROS_DEBUG}}
```

Comment:
I wish there was a better way to plumb this through! This is pretty verbose.

We could consider *always* adding `-Dsun.security.krb5.debug={{TASKCFG_ALL_SECURITY_KERBEROS_DEBUG}}` instead of the conditional mustache?